frontend/src/components/eam/components/tenant/tenant.tsx

```diff
@@ -0,0 +1,84 @@
+import { FC } from 'react';
+import { Button, Input } from 'components/common/common';
+import { InputType, ButtonType } from 'common/enums/enums';
+import {
+  useAppForm,
+  useAppDispatch,
+  useAppSelector,
+  useEffect,
+  useMemo,
+} from 'hooks/hooks';
+import { getNameOf } from 'helpers/helpers';
+import {
+  EAMTenantUpdateRequestDto,
+  EAMTenantFormDto,
+} from 'common/types/types';
+import { eamTenantUpdate as EAMTenantValidationSchema } from 'validation-schemas/validation-schemas';
+import { app as EAMTenantConfigurateActions } from 'store/actions';
+import { DEFAULT_PAYLOAD } from './common/constants';
+import styles from './styles.module.scss';
+
+const Tenant: FC = () => {
+  const dispatch = useAppDispatch();
+  const { tenant } = useAppSelector(({ app }) => ({
+    tenant: app.tenant,
+  }));
+
+  const defaultValues = useMemo(() => {
+    const values = {
+      ...DEFAULT_PAYLOAD,
+    };
+    if (tenant) {
+      values.name = tenant.name;
+    }
+    return values;
+  }, [tenant]);
```

Comment:
```suggestion
  const defaultValues = useMemo(() => mapTenantDefaultValues(tenant), [tenant]);
```
create this helper `mapTenantDefaultValues`